src/sagemaker_containers/_env.py

```diff
@@ -588,6 +601,14 @@ def additional_framework_parameters(self):  # type: () -> dict
         """
         return self._additional_framework_parameters
 
+    def sagemaker_s3_output(self):  # type: () -> str
+        """s3 output directory location provided by the user.
```

Comment:
S3 in docstring

Reply:
should this be a property?

Reply:
No, i don't want it to show up in the customer logs as available training env property.
I assume we will remove it when platform supports it.

Reply:
ok